docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"detailed_label_quality\"].head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db02e63d",
+   "metadata": {},
+   "source": [
+    "- `annotator_stats` which gives us the annotator quality score for each annotator, alongisde other information such as the number of examples each annotator labeled, their agreement with the consensus label and the class they perform the worst at. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "efe16638",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"annotator_stats\"].head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a0d09bfa",
+   "metadata": {},
+   "source": [
+    "The `annotator_stats` DataFrame is sorted by increasing `annotator_quality`, showing us the worst annotators first.\n",
+    "\n",
+    "Notice that in the above table annotators with ids 45 to 49 have the worst annotator quality score, which is expected because we made the last 5 annotators systematically worse than the rest."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20ca8dd2",
+   "metadata": {},
+   "source": [
+    "### Comparing improved consensus labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b49657d",
+   "metadata": {},
+   "source": [
+    "We can get the improved consensus labels from the `label_quality_multiannotator` DataFrame shown above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "abd0fb0b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "improved_consensus_label = multiannotator_dict[\"label_quality_multiannotator\"][\"consensus_label\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1fd7a5fd",
+   "metadata": {},
+   "source": [
+    "Since our toy dataset is synthetically generated by adding noise to each annotator's labels, we know the ground truth labels for each example. Hence we can compare the accuracy of the consensus labels obtained using majority vote, and the improved consensus labels obtained using cleanlab."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cdf061df",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "majority_vote_accuracy = np.mean(true_labels == majority_vote_label)\n",
+    "cleanlab_label_accuracy = np.mean(true_labels == improved_consensus_label)\n",
+    "\n",
+    "print(f\"Accuracy of majority vote labels = {majority_vote_accuracy}\")\n",
+    "print(f\"Accuracy of cleanlab consensus labels = {cleanlab_label_accuracy}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c20b2c9",
+   "metadata": {},
+   "source": [
+    "We can see that the accuracy of the consensus labels improved as a result of using cleanlab, which not only takes the annotators' labels into account, but also a model to compute better consensus labels."
+   ]
```

Comment:
For completeness sake, final (non-hidden) cell of notebook could show retraining the logistic regression model on the improved consensus labels?

Reply:
Is the purpose of this retraining to show how the user can use the improved model to predict test data and more? 
We are not trying to rerun our multiannotator functions again right (just confirming)

Reply:
No it's just to clarify what the purpose of the improved consensus labels is. I don't think we should even report accuracy of the retrained model or anything like that

Reply:
That said, after this extra cell you could (in text) just explain:  

We can repeatedly iterate this whole process of getting better consensus labels using the model's pred_probs and then retraining our model with the improved labels to get even better pred_probs!

Reply:
Not entirely sure I catch the best way to present this section, will be up in the new push and some feedback would be really helpful!